common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -0,0 +1,228 @@
+"""
+Acceptance tests for the Import and Export pages
+"""
+from abc import abstractmethod
+from bok_choy.promise import EmptyPromise
+from .base_studio_test import StudioLibraryTest, StudioCourseTest
+from ...fixtures.course import XBlockFixtureDesc
+from ...pages.studio.import_export import ExportLibraryPage, ExportCoursePage, ImportLibraryPage, ImportCoursePage
+from ...pages.studio.library import LibraryEditPage
+from ...pages.studio.container import ContainerPage
+from ...pages.studio.overview import CourseOutlinePage
+
+
+# pylint: disable=no-member
+class ExportTestMixin(object):
+    """
+    Tests to run both for course and library export pages.
```

Comment:
Can you add BDD descriptions to the test points, similar to https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py#L52